html/semantics/forms/form-submission-0/constructing-form-data-set.html

```diff
@@ -1,32 +1,49 @@
 <!DOCTYPE html>
-<head>
 <meta charset="utf-8">
 <link rel="help" href="https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#constructing-the-form-data-set">
 <link ref="help" href="https://xhr.spec.whatwg.org/#dom-formdata">
 <link rel="help" href="https://fetch.spec.whatwg.org/#concept-bodyinit-extract">
 <script src="/resources/testharness.js"></script>
 <script src="/resources/testharnessreport.js"></script>
-</head>
-<body>
 
-<iframe name="frame1"></iframe>
-<form accept-charset="iso-8859-1" target="frame1" action="/common/blank.html">
-<input type="hidden" name="_charset_">
+<iframe name="frame1" id="frame1"></iframe>
+<form accept-charset="iso-8859-1" target="frame1" action="/common/blank.html" id="form1">
+  <input type="hidden" name="_charset_">
+</form>
+
+<iframe name="frame2" id="frame2"></iframe>
+<form target="frame2" action="/common/blank.html" id="form2">
+  <input type="text" name="foo">
+  <button type="close" name="close" value="true">close</button>
+  <button type="button" name="button" value="true">button</button>
+  <button type="reset" name="reset" value="true">reset</button>
+  <button type="submit" name="submit" value="true">submit</button>
```

Comment:
Is it intentional to use "submit" as the name (since `form.submit()` exists)?

Reply:
It is nothing special, but when we use `form.submit()`, the submit button is not submitter, right? This is confirmed in the realization of Chrome and FireFox.